repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py

```diff
@@ -163,3 +163,34 @@ def mount_dnf_cache(overlayfs_info):
 
 def umount_dnf_cache(overlayfs_info):
     return check_cmd_call(['/bin/umount', os.path.join(overlayfs_info.merged, 'var', 'cache', 'dnf')])
+
+
+def create_disk_image(path):
+    diskimage_path = os.path.join(path, 'diskimage')
+    mounts_path = os.path.join(path, 'mounts')
+
+    if os.path.isdir(mounts_path):
+        check_cmd_call(['/bin/umount', '-fl', mounts_path])
```

Comment:
I know it is not related to this PR but do not we have already some stlib stuff for calling external programs?

Reply:
the function check_cmd_call should have been refactored in #13 to use stdlib.call but it wasn't so yeah this needs to be done in a different PR